sharktank/tests/evaluate/baseline_perplexity_scores.json

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
can we just store these values in a npy file or something and retrieve from sharkblobs?